### ISSUE TITLE
Getters/Setters

### ISSUE THREAD

**Issue by bvandenbon** [featurecat/lizzie#100]
I think it would be a big step forward to pause for a second and do some refacturing.

I don't want to be the person in the room that tells that we should follow strict coding rules,
because I am convinced that good software developers don't need a lot of rules. And I really do think that you are all good software developers. You're all GREAT !!! 😃 

However I do think we should be using getters and setters to access fields. Fields shouldn't be public (unless perhaps if they are static). But regular instance fields really shouldn't be public. -- The main problem with this, is that once you make fields public it's unclear if they are marked public just to be readable from outside the class, or if they are marked public to also be writeable from outside. Simply put, sometimes you want a getX(), sometimes you want a setX() and sometimes you want both. But with public fields you can't achieve this effect.

Actually I've been tempted a couple of times to just go in there and do the refacturing. However, my fear is that the person who will have to merge this will be really pissed if somebody just changes 30 files without any comments and without adding new functionality. So, that's why I wanted to discuss it in advance.

Looking forward to your opinions.
In all honesty, if you guys don't like this idea, then I'll just forget all about it.

**Comment by bittsitt** [comment]
I completely agree, and I don't think featurecat would be pissed. Refactoring is on his todo list in #64 . Now that there are no pending PRs is a very good time to do it, then it is easy to merge.

**Comment by featurecat** [comment]
Refactoring is a great idea. Maybe we should refactor directly after releasing 0.4 though. That way it's a little more relaxing. That would be in 1-2 days (hoping!). After release I'd like to refactor and set some guidelines for contributions. I'll merge any refactor PRs without anger :)

By the way I am a she :)
